packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -305,43 +312,145 @@ async function ensureAvailableUsers(
   return availableUsers;
 }
 
-async function handler(req: NextApiRequest & { userId?: number | undefined }) {
+function getBookingData({
+  req,
+  isNotAnApiCall,
+  eventType,
+}: {
+  req: NextApiRequest;
+  isNotAnApiCall: boolean;
+  eventType: Awaited<ReturnType<typeof getEventTypesFromDB>>;
```

Comment:
```suggestion
    args: { isNotAnApiCall: boolean;  eventType: Awaited<ReturnType<typeof getEventTypesFromDB>>; }
```

Reply:
I just don't want to pollute the function with extra arguments. I see these as ` args` or `options`